iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/hint/DocumentOrganizationBuilder.java

```diff
@@ -0,0 +1,51 @@
+package eu.dnetlib.iis.wf.affmatching.hint;
+
+import java.io.Serializable;
+
+import org.apache.spark.api.java.JavaPairRDD;
+import org.apache.spark.api.java.JavaRDD;
+
+import com.google.common.base.Preconditions;
+
+import eu.dnetlib.iis.wf.affmatching.model.DocumentOrganization;
+import eu.dnetlib.iis.wf.affmatching.model.DocumentProject;
+import eu.dnetlib.iis.wf.affmatching.model.ProjectOrganization;
+import scala.Tuple2;
+
+/**
+ * Creates {@link DocumentOrganization} objects based on {@link DocumentProject}
+ * and {@link ProjectOrganization} datastores.
+ * 
+ * @author mhorst
+ *
+ */
+public class DocumentOrganizationBuilder implements Serializable {
```

Comment:
Well, I am not sure if using Builder as part of the name is the best one here. Usually the name 'Builder' is used for classes that follow the https://en.wikipedia.org/wiki/Builder_pattern